### ISSUE TITLE
add filters to the Price and Quantity table headers.

### ISSUE THREAD

**Issue by richardtape** [eventespresso/event-espresso-core#84]
Have created a PR https://github.com/eventespresso/event-espresso-core/pull/83 to add extra filters to the ticket selector table headers (Available Tickets already had one, other columns did not).

**Comment by tn3rb** [comment]
thanks Richard,

I'll take a look

**Comment by richardtape** [comment]
:+1: Thank you!

There's possibly a few more coming as I start migrating us over to EE4. Will make them as generic as possible and fit into your coding styles, decisions and patterns.

**Comment by richardtape** [comment]
OK, I made a mess. Genuinely sorry. This time I have ensured I'm up-to-date with master (github simply forked something from aaaaages ago when I made my fork...weird). 

Not only that I'd made a mistake in the PR by having a _e() instead of __().

https://github.com/eventespresso/event-espresso-core/pull/85 fixes the above, can be straight merged and also adds your cherry picked esc_html__() call around the pre-existing filter, so with a bit of luck this should just be a straight replacement.

Again, really sorry... clearly not enough coffee this morning.
